test/minio/1.7.0/questions.yaml

```diff
@@ -252,6 +273,85 @@ questions:
                   type: hostpath
                   required: true
 
+  - variable: postgresAppVolumeMounts
```

Comment:
Should user be allowed to change host path for postgres once it has deployed already?

Reply:
Basically any change to underlying storage for an already initialized postgres?

Reply:
> Should user be allowed to change host path for postgres once it has deployed already?

No, not really. 

But if we remove this completely, users can't define a hostPath at all. It will always be stored in ix-volumes.

Other option is to write extra helm code, that only takes into account the path only when it's at an `isInstall` phase.
But then it gets messy real quick. You have to save the path defined the first time into a secret/configmap and retrieve it on each redeploy/upgrade. And also you will now have a path field that does nothing beyond the initial install.

Unless there is a way to lock (editable: false) the path field after the initial install.

Reply:
@stavros-k iirc the schema attr of interest in that case is `immutable` - that should cater to this use case. Also i think there are lots of other places in other apps which can benefit from the same change. So we can create a ticket to cater to those as well.

Reply:
Oh nice! 

Added `immutable` only on the postgres data path.
Not in the postgres backup path.

Backup path can be changed, without issues.